docs/user/logs.md

```diff
@@ -566,21 +566,21 @@ To detect and fix such situations, check the [pipeline status](./resources/02-lo
 **Symptom**:
 
 - The backend is reachable and the connection is properly configured, but some logs are refused.
-- In the LogPipeline status, the `TelemetryFlowHealthy` condition has status **SomeDataDropped**.
+- In the LogPipeline status, the `TelemetryFlowHealthy` condition has status **GatewaySomeTelemetryDataDropped** or **AgentSomeTelemetryDataDropped**.
 
 **Cause**: It can happen due to a variety of reasons - for example, the backend is limiting the ingestion rate.
 
 **Solution**:
 
-1. Check the `telemetry-log-gateway` and `telemetry-log-agent` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. Also, check your observability backend to investigate potential causes.
-2. If backend is limiting the rate by refusing logs, try the options desribed in [Gateway Buffer Filling Up](#gateway-buffer-filling-up).
+1. Check the `telemetry-log-gateway` Pod (in case of **GatewaySomeTelemetryDataDropped**) or `telemetry-log-agent` Pod (in case of **AgentSomeTelemetryDataDropped**) for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. Also, check your observability backend to investigate potential causes.
+2. If backend is limiting the rate by refusing logs, try the options desribed in [Buffer Filling Up](#gateway-buffer-filling-up).
 3. Otherwise, take the actions appropriate to the cause indicated in the logs.
```

Comment:
```suggestion
4. Otherwise, take the actions appropriate to the cause indicated in the logs.
```

Reply:
Why change it from 3 to 4?
It is indeed the 3rd item in the steps